src/System.Management.Automation/engine/DataStoreAdapterProvider.cs

```diff
@@ -37,14 +37,15 @@ public class ProviderInfo
         private SessionState _sessionState;
```

Comment:
@iSazonov, your last commit had 1 failures in `PowerShell-CI-windows`
Enter-PSHostProcess tests.By Process Id.Can enter using NamedPipeConnectionInfo
```powershell
Expected $true, but got $false.
at <ScriptBlock>, D:\a\1\s\test\powershell\Modules\Microsoft.PowerShell.Core\Enter-PSHostProcess.Tests.ps1: line 148
148:                     Should -BeTrue -Because (get-content $splat.FilePath -Raw)
```